<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_Equipement</class>
 <widget class="QDialog" name="Dlg_Equipement">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equipement</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="button_Ajouter">
        <property name="text">
         <string>&amp;Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_Supprimer">
        <property name="text">
         <string>&amp;Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_Modifier">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_Selectionner">
        <property name="text">
         <string>Selectionner</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_Fermer">
        <property name="text">
         <string>&amp;Fermer</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="gb_edit_champs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>255</height>
       </size>
      </property>
      <property name="title">
       <string>Ajouter un élement :</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>188</y>
         <width>87</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="button_Annuler">
          <property name="text">
           <string>A&amp;nnuler</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_Valider">
          <property name="text">
           <string>&amp;Valider</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>621</width>
         <height>231</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab_general">
        <attribute name="title">
         <string>Général</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>583</width>
           <height>111</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>N° série :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_NoSerie">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Offset :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Offset">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="ckB_EnService">
            <property name="text">
             <string>En service</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Min. Gamme :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_MinGamme">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Modèle :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_Modele">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_EditModele">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>45</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>45</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Editer</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Max. Gamme :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_MaxGamme">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_ParamCom">
        <attribute name="title">
         <string>Paramètres de communications</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>30</y>
           <width>601</width>
           <height>115</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Taux de transmission :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cb_Tx_Transmission"/>
              </item>
              <item>
               <widget class="QPushButton" name="button_EditTxTransmission">
                <property name="maximumSize">
                 <size>
                  <width>45</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Editer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <spacer name="horizontalSpacer_10">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Adresse :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_Adresse"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <spacer name="horizontalSpacer_11">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Nb bits de transmission :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cb_Nb_Bits_Transmission">
                <property name="currentIndex">
                 <number>3</number>
                </property>
                <item>
                 <property name="text">
                  <string>5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>6</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>7</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>8</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Nb bits de stop :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cb_Nb_Bits_Stop">
                <item>
                 <property name="text">
                  <string>1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>1.5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Type de controle de flux :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cb_Type_Controle_Flux">
                <item>
                 <property name="text">
                  <string>Aucun</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Hardware</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Xon/Xoff</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <spacer name="horizontalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Type de parité :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cb_Type_Parite">
                <item>
                 <property name="text">
                  <string>Aucune</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Odd</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Even</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mark</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Space</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Polluants associés</string>
        </attribute>
        <widget class="QPushButton" name="button_AddPolluant">
         <property name="geometry">
          <rect>
           <x>175</x>
           <y>70</y>
           <width>31</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_RemovePolluant">
         <property name="geometry">
          <rect>
           <x>175</x>
           <y>100</y>
           <width>31</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>161</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Polluants disponibles :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView_Molecule">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>10</y>
           <width>161</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Polluants associés :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView_PolluantsAssocies"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
